scripts/index_test_examples.py

```
@@ -0,0 +1,91 @@
+                'method_signatures': '\n'.join(method_signatures)
+            }
+            # Embed and add to ChromaDB
+            doc_text = f"SOURCE:\n{source_code}\n\nTEST:\n{test_code}"
+            vectorstore.add_texts([doc_text], metadatas=[metadata])
+            print(f"[RAG INDEX] Indexed test/source pair: {test_class_name} <-> {source_class_name}")
+        except Exception as e:
+            print(f"[RAG INDEX] Error processing {test_file}: {e}")
+
+    print("[RAG INDEX] Indexing complete.")
+
+def retrieve_similar_test_examples(class_code, method_signatures, vectorstore, top_n=3):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)